core/src/layers/otelmetrics.rs

```diff
@@ -69,196 +65,89 @@ impl OtelMetricsLayer {
     /// # async fn main() -> Result<()> {
     /// let meter = opentelemetry::global::meter("opendal");
     /// let op = Operator::new(services::Memory::default())?
-    ///     .layer(OtelMetricsLayer::builder().path_label(1).register(&meter))
+    ///     .layer(
+    ///         OtelMetricsLayer::builder()
+    ///             .bytes_boundaries(exponential_boundaries(1024f64, 2.0f64, 10))
```

Comment:
Users can pass `Vec<f64>` directly, we don't need to provide this helper functions.

Reply:
In fact, we can tell users that we provide a solid default configuration. They don't need to make any changes unless they have specific requirements.